project-example/world.json

```diff
@@ -1,5 +1,5 @@
 {
-  "template": "small",
+  "template": "w2_r0500_e5",
```

Comment:
I'm surprised that github doesn't pick up that this is already  merged in

Reply:
Possibly related to that this branch is out of date?